src/subsplit_dag.hpp

```diff
@@ -129,13 +129,14 @@ class SubsplitDAG {
   // Get Taxon's bitset clade positional id.
   size_t GetTaxonId(const std::string &name) const;
   // Get node based on node id.
-  SubsplitDAGNode *GetDAGNode(const size_t node_id) const;
+  SubsplitDAGNode GetDAGNode(const size_t node_id) const;
+  MutableSubsplitDAGNode GetDAGNode(const size_t node_id);
   // Get the node id based on the subsplit bitset.
   size_t GetDAGNodeId(const Bitset &subsplit) const;
   // Gets the node id of the DAG root.
   size_t DAGRootNodeId() const;
   // Return the node ids corresponding to the rootsplits.
-  const SizeVector &RootsplitIds() const;
+  ConstNeighborsView RootsplitIds() const;
```

Comment:
This appears in the public interface but isn't documented anywhere that I can find. What is it, and how can I use it? Just a few lines in this header file should be fine.

Reply:
I've added the doc comment on SubsplitDAGNode::GetEdge, maybe I should move it to the declaration of NeighborsView and add reference comments to all functions that return it?

Reply:
That sounds good. You don't need to refer to the docs in every function that returns it... just something at the top of the header file for subsplit DAG and subsplit DAG node would do. thanks! ✨

Reply:
I've moved the usage information to SubsplitDAGNode top section